BackEnd/funbox/src/auth/auth.controller.ts

```diff
@@ -0,0 +1,23 @@
+import { Body, Controller, Post } from '@nestjs/common';
+import { AuthService } from './auth.service';
+
+@Controller('auth')
+export class AuthController {
+    constructor(private authService: AuthService){}
+
+    @Post()
+    async checkJwt(@Body('accessToken') accessToken: string) {
+        return await this.authService.tokenValidation(accessToken);
+    }
```

Comment:
JwtStrategy를 만든 김에 여기서도 사용하면 좋을 것 같아요!

```suggestion
    @Post()
    @UseGuards(AuthGuard())
    checkJwt(@Req() req): UserAuthDto {
        return req.user;
    }
```